doc/rados/operations/pools.rst

```diff
@@ -188,7 +188,7 @@ consider removing them when you no longer need your pool::
 
 	ceph osd pool get {pool-name} crush_ruleset
 
-If the ruleset was "123", for example, you can check the other pools like so::
+If the ruleset ID was "123", for example, you can check the other pools like so::
```

Comment:
i think we are using "rulename" to reference the name of ruleset, and use "ruleset" to reference the id of it. see http://docs.ceph.com/docs/master/rados/operations/pools/#set-pool-values and http://docs.ceph.com/docs/master/rados/operations/crush-map/#crush-map-rules. if you happen to find ruleset id anywhere else in the doc, could you point me to it? thanks.